firebase/firebaseApp.js

```diff
@@ -0,0 +1,29 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { getAnalytics } from "firebase/analytics";
+// TODO: Add SDKs for Firebase products that you want to use
+// https://firebase.google.com/docs/web/setup#available-libraries
+
+// Your web app's Firebase configuration
+// For Firebase JS SDK v7.20.0 and later, measurementId is optional
+const firebaseConfig = {
```

Comment:
I guess we should create a `.env` file and add all these values there.
@pranshudobhal your views?

Reply:
Yes, we should add them in a `.env.local` file and add it to `.gitignore`.

We should also add the env variables in vercel.